monkey/infection_monkey/credential_collectors/mimikatz_collector/pypykatz_handler.py

```diff
@@ -19,6 +24,10 @@
 
 
 def get_windows_creds() -> List[WindowsCredentials]:
+    if not is_windows_os():
+        logger.debug("Skipping pypykatz because the operating system is not Windows")
+        return []
+
```

Comment:
`pypykatz_handler` is a boundary between mimikatz credential collection logic and pypykatz package. Checking whether it's running on windows is part of credential gathering logic, not an interface of pypykatz. This check should be done in `mimikatz_collector` or even in master, because master is delegating what to run